docs: add ui question on k8s faq (#50470)
gravitational/teleport: docs/pages/enroll-resources/kubernetes-access/faq.mdx

Guide](../auto-discovery/kubernetes/kubernetes.mdx) for more
documentation and examples.

## Does Teleport work with Kubernetes desktop UI applications?

Yes, Teleport generates a kubeconfig file (default `~/.kube/config`) when a user

> logins to a Kubernetes cluster. GUI tools such as Lens and other applications can 

Reviewer: ```suggestion
logs in to a Kubernetes cluster. GUI tools such as Lens can 
```
- Grammar fix ("logins to" is incorrect)
- Removing redundant wording ("such as ... and other applications")
